client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions-i2/pending-actions/style.scss

```diff
@@ -22,7 +22,16 @@
 
 	.pending-actions__action--success {
 		.pending-actions__action-icon {
-			fill: #008a20;
+			svg {
+				fill: #008a20;
+			}
+		}
+	}
+	.pending-actions__action--error {
+		.pending-actions__action-icon {
+			svg {
+				fill: #d94e4e;
```

Comment:
Same comment about color variables, I'm not sure this HEX has a corresponding value in our existing color variables; I would choose the closest equivalent from color-studio.blog

Reply:
TIL about color-studio.blog! 😍

Reply:
Fixed here [d866b58](https://github.com/Automattic/wp-calypso/pull/90593/commits/d866b58bddc9af36e6e2bf54cf36ff6bdbeb44f2)